docs/reference/primitives/rack/registry.md

```diff
@@ -0,0 +1,19 @@
+# Registry
+
+External Docker image repository.  When you install a [Rack](../rack) an internal Registry is created for use by the Rack.  You can add external private Registries to use when refering to private images in your [Build](../app/build.md)
```

Comment:
Let's simplify this a bit. What is the smallest single sentence you can use to describe what you get when you `convox registries add`